docs/options api.md

```diff
@@ -0,0 +1,203 @@
+# Archipelago Options API
+
+This document covers some of the generic options available using Archipelago's
+options handling system.
+
+For more information on where these options go in your world please refer to:
+ - [world api.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md)
```

Comment:
In another PR, you have relative markdown links (e.g. `/docs/world%20api.md` in #1192), but here it's the absolute path. If we're revamping docs, we should probably make sure they're all relative imports.